cumulus/client/bootnodes/src/schema/response.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto2";
+
+package response;
+
+message Response {
+    // Peer ID of the node on the parachain side.
+    required bytes peer_id = 1;
+
+    // Multiaddresses of the parachain side of the node. The list and format are the same as for
+	// the `listenAddrs` field of the `identify` protocol.
+    repeated bytes addrs = 2;
+
+    // Genesis hash of the parachain. Used to determine the name of the networking protocol
+	// to connect to the parachain. Untrusted.
+    required bytes genesis_hash = 3;
+
+    // So-called "fork ID" of the parachain. Used to determine the name of the networking protocol
+	// to connect to the parachain. Untrusted.
+    optional string fork_id = 4;
```

Comment:
dq: Shouldn't this be `bytes` similar to genesis_hash?

Reply:
No, `fork_id` is a string.